frontend/src/components/collective/CollectiveLedger.js

```diff
@@ -2,25 +2,27 @@ import React, { Component, PropTypes } from 'react';
 
 import CollectivePendingExpenses from './CollectivePendingExpenses';
 import CollectiveTransactions from './CollectiveTransactions';
-import CollectiveDonate from './CollectiveDonate';
 import Currency from '../Currency';
 
 export default class CollectiveLedger extends Component {
 
   render(){
-    const { collective, i18n, hasHost } = this.props; 
+    const { collective, i18n } = this.props; 
     return (
       <section id='budget' className='px2'>
-        <div className='CollectiveLedger clearfix md-flex'>
-          <div className='CollectiveLedgerItems'>
-            <div className='CollectiveLedgerAvailableFunds'>
-              <h4 className='Collective-title m0 -ff-sec -fw-bold'>{i18n.getString('fundsAvailable')}</h4>
-              <h3> <Currency value={ collective.balance/100 } currency={ collective.currency } precision={0} /></h3>
+        <div className='CollectiveLedger clearfix md-flex justify-center'>
+          <div className='CollectiveLedger-lhs clearfix flex-column mr2 max-width-2'>
+            <CollectivePendingExpenses {...this.props} /> 
```

Comment:
we should avoid `{...this.props}`, it's not a good practice as it makes it difficult to see what that component really needs.